security/pfSense-pkg-tinc/Makefile

```diff
@@ -0,0 +1,46 @@
+# $FreeBSD$
+
+PORTNAME=	pfSense-pkg-tinc
+PORTVERSION=	1.0.28
+CATEGORIES=	security
+MASTER_SITES=	# empty
+DISTFILES=	# empty
+EXTRACT_ONLY=	# empty
+
+MAINTAINER=	dinoex@FreeBSD.org
```

Comment:
dinoex@FreeBSD.org will not maintain pfSense package, you can set maintainership to coreteam@pfsense.org